pwndbg/gdblib/config.py

```diff
@@ -37,19 +37,17 @@
 # https://sourceware.org/gdb/onlinedocs/gdb/Parameters-In-Python.html
 class Parameter(gdb.Parameter):
     def __init__(self, param: pwndbg.lib.config.Parameter):
-        self.param = param
-
         # `set_doc` and `show_doc` must be set before `gdb.Parameter.__init__`.
         # They will be used for `help set <param>` and `help show <param>`,
         # respectively
-        self.set_doc = "Set " + self.param.docstring
-        self.show_doc = "Show " + self.param.docstring
-        super().__init__(self.param.name, gdb.COMMAND_SUPPORT, self._param_class())
-
-    def _param_class(self):
-        for k, v in PARAM_CLASSES.items():
-            if isinstance(self.param.value, k):
-                return v
+        self.set_doc = "Set " + param.docstring
+        self.show_doc = "Show " + param.docstring
+
+        param_class = PARAM_CLASSES[type(param.value)]
+        super().__init__(param.name, gdb.COMMAND_SUPPORT, param_class)
```

Comment:
Just checking, is it important that this comes before the assignment to `self.param` and `self.value` below? If not, this is fine, but if that's the case we should add a comment about it, similar to what I added above.

Reply:
I think it was, so i moved it back